Parsers/ASimNetworkSession/ARM/vimNetworkSessionMicrosoftWindowsFirewall/README.md

```diff
@@ -0,0 +1,15 @@
+# WindowsFirewall ASIM NetworkSessions Normalization Parser
```

Comment:
We still have the issue with the name. It can't be WindowsFirewall. Has to be "Windows Events Firewall" or something similar.